### PR TITLE
Wrap text node to reduce Google Translate bug

### PR DESCRIPTION
**Pull request by nedredmond**
[It looks like Google Translate doesn't play nicely with React.](https://martijnhols.nl/gists/everything-about-google-translate-crashing-react)
There is [a known issue that Google plans to address](https://issues.chromium.org/issues/41407169)... one day, maybe.
In the short term, it looks like if we wrap text nodes in elements, we should be good to go.

Issue: LEMS-2574

Testing:
Use Google Translate on any radio widget exercise and do something that causes the radio widgets to change state to render a text node (the easiest way was showing rationales). The red bar of doom doesn't show.

### CODE REVIEW

**Comment by jeremywiebe** [comment]
Hrm. I've looked at this Sentry issue many times and had no idea where it came from. I wonder if it's all down to Google Translate or other DOM manipulation tooling? 

https://khanacademyorg.sentry.io/issues/4737185317/?project=15744&query=is%3Aunresolved%20Perseus&referrer=issue-stream&statsPeriod=14d&stream_index=0

**Comment by nedredmond** [comment]
@jeremywiebe seems likely!

File: packages/perseus/src/widgets/radio/choice-icon/choice-icon.tsx [hunk 1]
```diff
@@ -68,14 +68,14 @@ function getDynamicStyles(
...
     borderColor: string;
     color: string;
     borderRadius: number;
 } {
-    let backgroundColor;
-    let borderColor;
-    let color;
+    let backgroundColor: string | undefined;
+    let borderColor: string;
+    let color: string;
```
Comment (benchristel) [inline packages/perseus/src/widgets/radio/choice-icon/choice-icon.tsx:78]: Thanks for fixing up these type issues!

**Review by benchristel — APPROVED** [review]
LGTM, thanks!

**Review by mark-fitzgerald — APPROVED** [review]
Ooof. Adding elements "just because" makes me sad.

Thanks for finding a simple work-around!

**Pull request merged**